src/Serializers/FingerprintSerializer.php

```diff
@@ -47,25 +53,22 @@ public function addIdToSerialization( Entity $entity, array &$serialization ) {
 	}
 
 	public function addLabelsToSerialization( Entity $entity, array &$serialization ) {
-		$labels = $entity->getLabels();
+		$labels = $entity->getFingerprint()->getLabels();
```

Comment:
Actual means here the one that contains the necessary information. $entity->getLabels(); just creates a new array with a subset of the information which is not enough here.